cfbs/commands.py

```diff
@@ -401,19 +402,47 @@ def _someone_needs_me(this) -> bool:
     return 0
 
 
-@commit_after_command("Updated all modules")
-def update_command():
-    new_deps = []
-    new_deps_added_by = dict()
+def update_command(to_update):
     config = CFBSConfig.get_instance()
     index = config.index
+
+    if to_update:
+        to_update = [Module(m) for m in to_update]
+        index.translate_aliases(to_update)
+        skip = (
+            m for m in to_update if all(n["name"] != m.name for n in config["build"])
+        )
+        for m in skip:
+            log.warning("Module '%s' not in build. Skipping its update.", m.name)
+            to_update.remove(m)
+    else:
+        # Update all modules in build if no modules are specified
+        to_update = [Module(m["name"]) for m in config["build"]]
+
+    new_deps = []
+    new_deps_added_by = dict()
     changes_made = False
+    updated = []
+
     for module in config["build"]:
+        update = None
+        for m in to_update:
+            if m.name == module["name"]:
+                update = m
```

Comment:
This logic where there's a list of things to update, but then the iteration goes over all modules instead of the things to update is a bit twisted, IMHO. I'd expect something like this:
```
for update_req in to_update:
    name = get_name_from_update_req(update_req)
    module = config["build"].get(name)
    ...
```